docs/inconsistent_url_paths.md

```diff
@@ -0,0 +1,9 @@
+---
+id: "inconsistent_url_paths"
+title: "Inconsistent URL Paths"
+sidebar_label: "Inconsistent URL Paths"
+---
+
+The [official Go implementation](https://github.com/twitchtv/twirp) differs in behavior from what is described in the [specification](https://twitchtv.github.io/twirp/docs/spec_v5.html). It modifies the service and method names to be CamelCase (with an initial capital) instead of using the exact names specified in the protobuf definition. This means that the URL paths generated for Go clients and servers may differ from paths generated for other language implementations. This issue is discussed in [#244](https://github.com/twitchtv/twirp/issues/244).
```

Comment:
```suggestion
The [official Go implementation](https://github.com/twitchtv/twirp) differs in behavior from the [v5 specification](https://twitchtv.github.io/twirp/docs/spec_v5.html) it how it handles URL Paths. Twirp Go modifies the service and method names to be CamelCase (with an initial capital) instead of matching the exact names specified in the protobuf definition. For example, if the proto file defines `service _my_service_2`, the route will instead match `XMyService_2` with routes like `/twirp/my.pkg.XMyService_2/MyMethod`. This means that the URL paths generated for Go clients and servers may differ from paths generated for other language implementations. This issue is discussed in [#244](https://github.com/twitchtv/twirp/issues/244).
```